kubejs/server_scripts/base/recipes/blue_skies/shaped.js

```diff
@@ -0,0 +1,117 @@
+ServerEvents.recipes((event) => {
+    const id_prefex = 'enigmatica:base/blue_skies/shaped/';
+
+    const recipes = [
+        {
+            output: 'minecraft:stonecutter',
+            pattern: [' A ', 'BBB'],
+            key: {
+                A: '#forge:ingots/ventium',
+                B: '#blue_skies:stone'
+            },
+            id: `blue_skies:stonecutter_compat`
+        },
+        {
+            output: 'blue_skies:warding_pearl',
+            pattern: ['AAA', 'BCB', 'AAA'],
+            key: {
+                A: '#forge:nuggets/falsite',
+                B: 'blue_skies:soul_fragment',
+                C: 'blue_skies:pearl'
+            },
+            id: `blue_skies:warding_pearl`
+        },
+        {
+            output: 'blue_skies:ventium_shears',
+            pattern: ['A  ', ' A '],
```

Comment:
Could this not be `['A ', ' A']` instead? Should give a few more placement options

Reply:
Ah, good point. Will fix that up quick